src/lando/dockerflow/decorators.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import time
+from functools import wraps
+
+# Logging is not yet implemented
+request_logger = logging.getLogger("dockerflow.request_summary.logger")
+
+
+def log_request(view):
+    @wraps(view)
+    def _wrapped_view(self, request, *args, **kwargs):
+        start_time = time.time()
+        response = view(self, request, *args, **kwargs)
+        end_time = time.time()
+
+        summary = {
+            "errno": 0 if response.status_code < 400 else 1,
+            "agent": request.headers.get("User-Agent", ""),
+            "lang": request.headers.get("Accept-Language", ""),
+            "method": request.method,
+            "path": request.path,
+            "code": response.status_code,
+            "t": int(1000 * (end_time - start_time)),
+        }
```

Comment:
Technically this is an [optional recommendation](https://github.com/mozilla-services/Dockerflow?tab=readme-ov-file#optional-recommendations), I think we could probably punt this to when logging is more robust (probably as part of #12)

Reply:
Alright, I'll leave it as is.